pom.xml

```diff
@@ -319,7 +319,8 @@
         <slf4j.version>1.7.36</slf4j.version>
         <metrics.version>3.2.6</metrics.version>
         <mockito.version>4.11.0</mockito.version>
-        <zookeeper.version>3.5.9</zookeeper.version>
+        <zookeeper.version>3.8.2</zookeeper.version>
```

Comment:
The Zookeeper community has released 3.9.0. I get that 3.8.2 is still a stable release, but just getting the thoughts to adopt the latest .0 release.

Reply:
I updated to 3.9.0 - let's see, if the build is ok with the change.